### PR TITLE
An 5156/jupv4 v5 swaps logs

### PR DESCRIPTION
**Pull request by tarikceric**
Create inner swaps table for jupv5 using decoded logs

In upcoming PR: add view on top of silver table and update core tables

Note: 12 txs missing from the first day the program was live (2023-05-30)as logs could not be decoded for those
```
with missing as (
select * from  solana.silver.swaps_intermediate_jupiterv5_1
where tx_id not in (select tx_id from solana_dev.silver.swaps_inner_intermediate_jupiterv5
where aggregator_program_id = 'JUP5pEAZeHdHrLxh5UCwAbpjGwYKKoquCpda2hfP4u8')
)

-- select * from missing;

select * from solana.silver.decoded_logs
where tx_id in (select tx_id from missing)
and block_timestamp::date = '2023-05-30'
and program_id = 'JUP5pEAZeHdHrLxh5UCwAbpjGwYKKoquCpda2hfP4u8';
```

Test:
`05:42:42  Finished running 19 data tests, 11 project hooks in 0 hours 0 minutes and 42.64 seconds (42.64s).`

FR:
`04:35:00  2 of 2 OK created sql table model silver.swaps_inner_intermediate_jupiterv5 .... [SUCCESS 1 in 30.13s]`

### CODE REVIEW

File: models/silver/swaps/jupiter/v5/silver__swaps_inner_intermediate_jupiterv5.sql [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        {{ ref('silver__decoded_logs') }}
+    WHERE
+        program_id IN (
+            'JUP5pEAZeHdHrLxh5UCwAbpjGwYKKoquCpda2hfP4u8',
+            'JUP5cHjnnCx2DppVsufsLrXs8EBZeEZzGtEK9Gdz6ow'
+        )
+        AND event_type = 'SwapEvent'
+        AND succeeded
+        AND _inserted_timestamp :: DATE BETWEEN '2024-09-06'
+        AND '2024-09-07'
```
Comment (desmond-hui) [inline models/silver/swaps/jupiter/v5/silver__swaps_inner_intermediate_jupiterv5.sql:40]: nit: Should be 1 line

File: models/silver/swaps/jupiter/v5/silver__swaps_inner_intermediate_jupiterv5.yml [hunk 2]
```diff
@@ -0,0 +1,97 @@
+version: 2
+models:
+  - name: silver__swaps_inner_intermediate_jupiterv5
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - TX_ID
+            - INDEX
+            - LOG_INDEX
```
Comment (desmond-hui) [inline models/silver/swaps/jupiter/v5/silver__swaps_inner_intermediate_jupiterv5.yml:9]: LOG_INDEX is missing column description and tests

**Review by desmond-hui — CHANGES_REQUESTED** [review]

File: models/silver/swaps/jupiter/v5/silver__swaps_inner_intermediate_jupiterv5.sql (hunk 1 above)
Comment (tarikceric) [reply models/silver/swaps/jupiter/v5/silver__swaps_inner_intermediate_jupiterv5.sql]: updated

File: models/silver/swaps/jupiter/v5/silver__swaps_inner_intermediate_jupiterv5.yml (hunk 2 above)
Comment (tarikceric) [reply models/silver/swaps/jupiter/v5/silver__swaps_inner_intermediate_jupiterv5.yml:9]: added and tests passed

16:03:14  Finished running 1 table model, 11 project hooks in 0 hours 1 minutes and 26.53 seconds (86.53s).
16:03:14  
16:03:14  Completed successfully
16:03:14  
16:03:14  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1

**Review by desmond-hui — APPROVED** [review]

**Pull request merged**